help_function/src/onedpl_test_transform_output_iterator.cpp

```diff
@@ -0,0 +1,441 @@
+// ====------ onedpl_test_transform_output_iterator.cpp-- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+
+#include "oneapi/dpl/algorithm"
+#include "oneapi/dpl/execution"
+#include "oneapi/dpl/iterator"
+
+#include "dpct/dpct.hpp"
+#include "dpct/dpl_utils.hpp"
+
+#include <CL/sycl.hpp>
+
+#include <iomanip>
```

Comment:
Maybe I missed it, but is the `iomanip` header used anywhere?

Reply:
nope, removed.